redfish-core/include/registries/openbmc_message_registry.hpp

```diff
@@ -2434,6 +2434,17 @@ constexpr std::array<MessageEntry, 189> registry = {
             {"string"},
             "None.",
         }},
+    MessageEntry{
+        "HardwareIsolationReason",
```

Comment:
Can we push this part upstream? 
Aren't these alphabetical ?

Reply:
> Can we push this part upstream?

I think no since our guard feature is downstream. If you are ok, I can push into upstream (no reference for the backend code in the upstream). 

> Aren't these alphabetical ?

Ah, it is my mistake (did not realize that message registry is defined in the alphabetical order) I will fix it

Reply:
>> Aren't these alphabetical ?

> Ah, it is my mistake (did not realize that message registry is defined in the alphabetical order) I will fix it

Done - Added the new commit `Fixup` (it can be merged into the previous commit without commit message) as per the latest PR process.